### PR TITLE
Load config files from URL

### PR DESCRIPTION
**Pull request by prete**
Keep the `index.html?config=http://exmaple.com/config.json` option so we can load config files from URL. 
This is important beacuse we can discovers bugs/defects, showcase the app or try new layouts before the release and don't have to depend on http://vitessce.io or other deployments out there.

### CODE REVIEW

File: src/Config.js [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+    var configJSON = {};
+    try {
+        if (isValidURL(config)) {
+            configJSON = loadConfigFromURL(config);
+        } else {
+            configJSON = loadConfigFromLocal(config);
+        }
+        if (!config.version) {
+            // add version property if it's missing
+            config.version = "X.Y.Z";
```
Comment (davehorsfall) [inline src/Config.js:29]: Just out of interest @prete, what is the motivation for checking this property exists? Is it needed somewhere?

**Comment by davehorsfall** [comment]
@prete I've just pushed a slightly refactored branch that no longer loads config files locally. I've moved the sample configs into the public folder where they can be loaded directly using the `config=` parameter. 

Have a look, let me know what you think.

**Comment by prete** [comment]
> Please note, the config= url parameter is still working in the latest release. 

I did a build over the weekend and I can assure it wasn't working.

> what is the motivation for checking this property exists? Is it needed somewhere?

Annoyingly, it is [here](https://github.com/vitessce/vitessce/blob/7e08d6b26c8aa2724fe3b684480fa8da930aad16/src/app/Vitessce.js#L69)

Just a heads up, this won't work for getting the default config, because _index.html_ will get in the way:
```javascript
return window.location.protocol + '//' + window.location.host + window.location.pathname + '/config/iss-human-brain-advanced.json';
```

For the current path you'll get `https://hindlimb.cog.sanger.ac.uk/spatial-web-app/latest/index.html/config/iss-human-brain-advanced.json`. What you want is:

```javascript
window.location.protocol + '//' + window.location.host + window.location.pathname.substring(0, window.location.pathname.lastIndexOf('/'))
```

**Comment by davehorsfall** [comment]
> I did a build over the weekend and I can assure it wasn't working.

Strange. I think maybe the branches were out of sync, or there was some other problems. I aligned all the branches last night. Could you please try building again with this `config-from-url` branch and let me know if it works?

> Annoyingly, it is [here](https://github.com/vitessce/vitessce/blob/7e08d6b26c8aa2724fe3b684480fa8da930aad16/src/app/Vitessce.js#L69)

Ok, I've included the check in the `App.js` which fetches the remote config. 

> Just a heads up, this won't work for getting the default config, because _index.html_ will get in the way:

Thanks. Didn't spot that. VScode automatically loads server without index.html which is why it was working for me. I've used the relative path to simply things as much as possible.

**Pull request merged**